steps/mentor-calibre-lvs/configure.yml

```diff
@@ -13,6 +16,7 @@ name: mentor-calibre-lvs
 
 inputs:
   - adk
+  - adk_lvs2
```

Comment:
This is AHA-specific and should be done dynamically in our graph, as opposed to statically baked into the library node.

So use .extend_inputs() like this -- https://github.com/StanfordAHA/garnet/blob/master/mflowgen/Tile_PE/construct.py#L123

Reply:
> This is AHA-specific and should be done dynamically in our graph, as opposed to statically baked into the library node.
> 
> So use .extend_inputs() like this -- https://github.com/StanfordAHA/garnet/blob/master/mflowgen/Tile_PE/construct.py#L123

RIGHT! extend_inputs(). I forgot about extend_inputs(). That's exactly what I was wanting. Thanks!